packages/ubuntu_wizard/lib/l10n.dart

```diff
@@ -52,12 +52,15 @@ class LocalizedLanguage {
 /// [locales] must contain the base locale i.e. the template .arb locale.
 Future<List<LocalizedLanguage>> loadLocalizedLanguages(
     List<Locale> locales) async {
-  assert(locales.contains(_kBaseLocale));
```

Comment:
I left this out because it wasn't really important and the base locale is "en_US", but AppLocalizations.supportedLocales no longer contains full locales (because .arb has only the country code). We extract the full locale below instead.